hysds/dataset_ingest_bulk.py

```diff
@@ -0,0 +1,946 @@
+from __future__ import division
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import absolute_import
+
+from builtins import str
+from builtins import int
+from builtins import open
+from future import standard_library
+
+standard_library.install_aliases()
+
+import os
+import re
+import json
+import requests
+import backoff
+import shutil
+import traceback
+
+from glob import glob
+from datetime import datetime
+from subprocess import check_output
+
+from io import StringIO
+from lxml.etree import parse
+from urllib.parse import urlparse
+from importlib import import_module
+
+from tempfile import mkdtemp
+
+from hysds.utils import get_disk_usage, makedirs, get_job_status, dataset_exists, find_dataset_json
+from hysds.log_utils import logger, log_prov_es, log_custom_event, log_publish_prov_es, backoff_max_value, \
+    backoff_max_tries
+from hysds.celery import app
+from hysds.recognize import Recognizer
+from hysds.orchestrator import do_submit_job
+
+import osaka.main
+
+
+FILE_RE = re.compile(r"file://(.*?)(/.*)$")
+SCRIPT_RE = re.compile(r"script:(.*)$")
+BROWSE_RE = re.compile(r"^(.+)\.browse\.png$")
+
+
+class NoDedupJobFoundException(Exception):
+    def __init__(self, message):
+        self.message = message
+        super(NoDedupJobFoundException, self).__init__(message)
+
+
+class NoClobberPublishContextException(Exception):
+    pass
+
+
+class NotAllProductsIngested(Exception):
+    pass
+
+
+def get_module(m):
+    """Import module and return."""
+
+    try:
+        return import_module(m)
+    except ImportError:
+        logger.error('Failed to import module "%s".' % m)
+        raise
+
+
+def get_func(f):
```

Comment:
@DustinKLo: would we be able to use `get_func` from `hysds.utils`?

Reply:
good catch, `get_func` still exists in `hysds.utils` so ill remove it here